### DISCUSSION TITLE
How can i autoimport

### DISCUSSION THREAD

**Discussion by mitraecp** [nuxt/nuxt#17054] [Questions]
How can i autoimport interfaces from a folder?

![image](https://user-images.githubusercontent.com/75841640/184879701-90e7371f-a7d0-47d9-9dd3-5c63f2cdc09b.png)

**Comment by jf908** [comment]
You can use TypeScript's declaration feature for this. I've created a file in `/types/app.d.ts` with
```d.ts
declare namespace App {
  export interface MyInterface {
    foo: string;
  }
}
```
and it is globally available as `App.MyInterface`.
